web-registry/src/components/molecules/BlockContentBody/BlockContentBody.tsx

```diff
@@ -0,0 +1,20 @@
+import { BlockContent } from 'web-components/lib/components/block-content';
+import { Body } from 'web-components/lib/components/typography';
+
+import { Scalars } from 'generated/sanity-graphql';
+
+export interface Props {
+  body: Scalars['JSON'];
+}
+
+const BlockContentBody = ({ body }: Props): JSX.Element => (
+  <Body
+    size="xl"
+    textAlign="center"
+    sx={{ margin: '0 auto', maxWidth: 800, pb: { xs: 12.5, sm: 12.5 } }}
```

Comment:
still waiting for confirmation whether we want the same pb on mobile/desktop or not